### ISSUE TITLE
[1.7.10]: Need to fix errors with Entity Culling Unofficial

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#461]
### Modpack

Biggess Pack Cat Edition V0.85.71NOTSERVERCOMPATIBLE

### Game log

[fml-client-latest.log](https://github.com/quentin452/privates-minecraft-modpack/files/12448744/fml-client-latest.log)

### Description

Typical error : 
```
[22:05:23] [CullThread/INFO] [STDERR/entityculling]: [java.lang.Throwable$WrappedPrintStream:println:749]: java.lang.IndexOutOfBoundsException: bitIndex < 0: -2147483648
[22:05:23] [CullThread/INFO] [STDERR/entityculling]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.util.BitSet.set(BitSet.java:444)
[22:05:23] [CullThread/INFO] [STDERR/entityculling]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.logisticscraft.occlusionculling.OcclusionCullingInstance.isAABBVisible(OcclusionCullingInstance.java:92)
[22:05:23] [CullThread/INFO] [STDERR/entityculling]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.tclproject.entityculling.CullTask.run(CullTask.java:85)
[22:05:23] [CullThread/INFO] [STDERR/entityculling]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.Thread.run(Thread.java:750)
```

To reproduce the issue , only join world and wait some seconds

**Comment by quentin452** [comment]
This bug can appear with only Entity Culling enabled

due to"java.lang.NullPointerException: Invalid Biome id"  crash(when moving) caused by all mods removed but not Entity Culling , you need to don't move

1 : download the world
2 : open world , don't move
3 : wait for errors

download the world : https://mega.nz/file/utc2EAAT#6FcEXi4sn3L9_K36wBajL3ywTwDzcY8YSLbLDR4RaB4

**Comment by quentin452** [comment]
fixed in V0.85.73NOTSERVERCOMPATIBLE by updating entity culling
